### ISSUE TITLE
Pointless parameter in kMeans-clustering

### ISSUE THREAD

**Issue by Letifery** [Vincent-Fuecks/income-analysis-USA-Unsupervised-Learning#5]
**Problem**: The parameter n_init is useless since it simulates our strat_kfold_evaluation() function and increases the runtime of an iteration by an absurd amount (kMeans is called in our strat_kfold_evaluation() function, thus results in fold*n_init[i] loops/kMean() callings per iteration). Plotting the accuracies of both algorithms also shows that changes in n_init yields almost no increase in accuracy.

![download](https://user-images.githubusercontent.com/34723172/148895626-7acacaa2-57ab-48d3-bbe7-eb7f7f66cdf0.png)

**Solution**: This parameter should be removed and replaced with n_clusters/init. n_init should be set to 1 in the function call itself 

**Reference**: https://scikit-learn.org/stable/modules/generated/sklearn.cluster.KMeans.html
